buildconfig/CMake/FindSphinx.cmake

```diff
@@ -12,7 +12,7 @@
 #=============================================================
 
 find_program( SPHINX_EXECUTABLE NAME sphinx-build.cmd sphinx-build
-  PATHS ${CMAKE_LIBRARY_PATH}/python2.7/Scripts
+  PATHS ${CMAKE_LIBRARY_PATH}/python${PYTHON_MAJOR_VERSION}.${PYTHON_MINOR_VERSION}/Scripts
```

Comment:
I don't think this works. The problem is the executable name is different, also depending on OS. I see `sphinx-build-3` on Fedora, `sphinx-build3` on Arch, and for Ubuntu it's located at `/usr/share/sphinx/scripts/python3/sphinx-build`.

But this is a different issues that can be fixed at another time.